Formula/fortio.rb

```diff
@@ -23,7 +23,8 @@ class Fortio < Formula
   depends_on "go" => :build
 
   def install
-    system "make", "official-build-clean", "official-build-version", "OFFICIAL_BIN=#{bin}/fortio"
+    system "make", "official-build-clean", "official-build-version", "OFFICIAL_BIN=#{bin}/fortio",
+	"BUILD_DIR=./tmp/fortio_build"
```

Comment:
Maybe something like:

```suggestion
    system "make", "official-build-internal", "OFFICIAL_BIN=#{bin}/fortio", "BUILD_DIR=#{buildpath}/build"
```

Or some other directory name if `build` is already used. I didn't see anything that needed the extra nesting.

`#{buildpath}` should be absolute path to current PWD (top of cloned source tree).

Reply:
./tmp works fine (and is in .gitignore) it's just that earlier the bot didn't like the line length and now doesn't like the tab